extensions/logging/src/__tests__/fixtures/fluentd.docker.ts

```diff
@@ -0,0 +1,63 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/extension-logging
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {expect} from '@loopback/testlab';
+import * as path from 'path';
+import {GenericContainer, StartedTestContainer} from 'testcontainers';
+
+export const ROOT_DIR = path.join(__dirname, '../../../fixtures');
+export const ETC_DIR = path.join(ROOT_DIR, 'etc');
+
+async function startFluentd() {
+  const container = await new GenericContainer(
+    'fluent/fluentd',
+    'v1.7.4-debian-1.0',
+  )
```

Comment:
Please don't start Docker containers from the tests. It slows down the test suite that's already too slow. If our tests need an external service to be running, then please provide a script that the developers can run ONCE to start all services at the beginning of their coding session. See what we have already in place in `acceptance/repository-*` directories and in individual connectors like [loopback-connector-mysql](https://github.com/strongloop/loopback-connector-mysql/blob/9860f0339f171423c4260ae999cabcc36cb0c1ce/setup.sh).

As we discussed when working on the monitoring/Prometheus integration, it's often a good practice to mock the external dependency (i.e. the log collector) for integration tests to make them faster and then write only  few acceptance/end-to-end tests that are talking to the real service started in Docker.